### DISCUSSION TITLE
MDAnalysis/MolSSI Workshop at ASU teaches MDAkits (and more)

### DISCUSSION THREAD

**Discussion by orbeckst** [MDAnalysis/mdanalysis#4597] [MDAKit Discussions]
If you're interested in learning how turn your MDAnalysis-based code into a package that others can use easily, check out the join MDAnalysis/MolSSI workshop at Arizona State University. See the post https://github.com/MDAnalysis/mdanalysis/discussions/4596

**Comment by RMeli** [comment]
Closing since the registration deadline has passed.
